.github/actions/ortdocker/check_image.py

```diff
@@ -47,4 +52,4 @@
     if version in versions:
         print("found")
     else:
-        print("none")
+        print(version)
```

Comment:
Is this change intended, and does it belong to the topic of this commit?

Reply:
Please explain this here instead of just marking my comment as "resolved".

Reply:
Did i not explain one line above ?